install_config/install/advanced_install.adoc

```diff
@@ -340,6 +340,31 @@ default is
 If you alter this variable, ensure the host name is accessible via your router.
 |===
 
+[[advanced-install-configuring-registry-location]]
+=== Configuring a Registry Location
+
+If you are using an image registry other than the default at
+`registry.access.redhat.com`, specify the desired registry within the
```

Comment:
Should this be formatted as *_registry.access.redhat.com_* since it is technically a file path in that it is a URL? I would have to see how we treat that elsewhere in the docs. I could be wrong.